packages/consumption/src/consumption/ConsumptionCoreErrors.ts

```diff
@@ -310,6 +310,10 @@ class Requests {
         return new ApplicationError("error.consumption.requests.validation.inheritedFromItem", message);
     }
 
+    public cannotShareARequestWithYourself() {
```

Comment:
I think usually we don't have an "a" in the name of errors e.g. we have cannotShareRelationshipAttributeOfPendingRelationship above and not cannotShare**A**RelationshipAttributeOf**A**PendingRelationship. So, I'd rather name it "cannotShareRequestWithYourself". But that's just a detail, I'm not sure how strict we are with this in general.